pkg/kubelet/server/stats/types.go

```diff
@@ -45,6 +45,8 @@ type NodeStats struct {
 	Memory *MemoryStats `json:"memory,omitempty"`
 	// Stats pertaining to network resources.
 	Network *NetworkStats `json:"network,omitempty"`
+	// Stats pertaining to total usage of filesystem resources.
+	Fs []FsStats `json:"fs,omitempty"`
```

Comment:
Yes with FsStats not device-specific I think this makes sense.  It will make future support for supporting multiple devices at the node level a bit ugly.  @vishh Can we say that we have no future plans to support nodes with multiple devices?